### DISCUSSION TITLE
请帮我看下, demo 文件夹下的md文件没生效

### DISCUSSION THREAD

**Discussion by vaynevayne** [umijs/dumi#1664] [Q&A] [answered]
这是antd 的
<img width="1199" alt="image" src="https://github.com/umijs/dumi/assets/124666577/77952cda-6763-44ce-b6d0-8628024b5b2c">

这是我的
<img width="1144" alt="image" src="https://github.com/umijs/dumi/assets/124666577/ddc32d8f-3f79-4df2-b8e2-e4daba9780da">
<img width="1167" alt="image" src="https://github.com/umijs/dumi/assets/124666577/59598d14-2ef7-46e9-af9b-11855309d2b6">

**Comment by PeachScript** [comment, marked as answer]
antd 的用法是为了向前兼容通过插件定制过的，不具备参考价值；demo 写法以 dumi 文档为准：https://d.umijs.org/guide/write-demo#%E6%A0%87%E9%A2%98%E4%B8%8E%E7%AE%80%E4%BB%8B

**Reply by vaynevayne** [reply]
多谢，看懂了
